app/views/other_change_validation_requests/_form.html.erb

```diff
@@ -37,15 +37,27 @@
         </div>
       <% end %>
 
-      <%= form.govuk_text_area :summary,
-                               label: { size: "s", class: "govuk-label govuk-label--s"},
-                               rows: 5,
-                               readonly: @planning_application.validated? %>
+      <%= form.govuk_text_area(
+        :summary,
+        label: {
+          size: "s",
+          class: "govuk-label govuk-label--s",
+          text: (t(".labels.summary") if @validate_fee)
```

Comment:
This form is a generic one for `OtherChangeValidationRequest`. When `@validate_fee` is true (ie the change is applied to the fee) we need to override the generic labels and hints.